2.illumination_correction/1.evaluate_qc.ipynb

```diff
@@ -0,0 +1,875 @@
+      "DAPI    3\n",
+      "Name: Channel, dtype: int64\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Metadata_Plate</th>\n",
+       "      <th>Metadata_Site</th>\n",
+       "      <th>Metadata_Well</th>\n",
+       "      <th>ImageQuality_PowerLogLogSlope</th>\n",
+       "      <th>ImageQuality_PercentMaximal</th>\n",
+       "      <th>Channel</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>768</th>\n",
+       "      <td>slide1</td>\n",
+       "      <td>M60</td>\n",
+       "      <td>A3</td>\n",
+       "      <td>-2.530333</td>\n",
+       "      <td>0.000019</td>\n",
+       "      <td>DAPI</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>659</th>\n",
+       "      <td>slide2</td>\n",
+       "      <td>M57</td>\n",
+       "      <td>B4</td>\n",
+       "      <td>-2.534797</td>\n",
+       "      <td>0.000019</td>\n",
+       "      <td>DAPI</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>615</th>\n",
+       "      <td>slide2</td>\n",
+       "      <td>M17</td>\n",
+       "      <td>B4</td>\n",
+       "      <td>-2.583345</td>\n",
+       "      <td>0.402079</td>\n",
+       "      <td>DAPI</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    Metadata_Plate Metadata_Site Metadata_Well  ImageQuality_PowerLogLogSlope  \\\n",
+       "768         slide1           M60            A3                      -2.530333   \n",
+       "659         slide2           M57            B4                      -2.534797   \n",
+       "615         slide2           M17            B4                      -2.583345   \n",
+       "\n",
+       "     ImageQuality_PercentMaximal Channel  \n",
+       "768                     0.000019    DAPI  \n",
+       "659                     0.000019    DAPI  \n",
+       "615                     0.402079    DAPI  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Calculate Z-scores for the column\n",
+    "z_scores = zscore(df['ImageQuality_PowerLogLogSlope'])\n",
+    "\n",
+    "# Identify outlier rows based on Z-scores below the mean since we are looking for the blurriest images (more negative)\n",
```

Comment:
I think this will have to be a MUST for other projects. In this project, unfortunately, this dataset is really weird, and what is considered blurry isn't very blurry at all (mainly containing large artifacts). 

This will be beyond the scope of this PR but I will need to keep this in mind for other projects. Plus I need practice with IPython anyway 😄